src/blocks/asha03/meta.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This folder neither exists in this PR nor on the master branch as far as I can see. Please remove it to avoid breaking the GitHub Actions workflow.